.github/workflows/ci_docker.yml

```diff
@@ -73,6 +77,13 @@ jobs:
         env:
           CI: false
 
+      # Check if only the workflow file is changed.
+      - name: Check workflow diff
+        id: check-workflow-diff
+        uses: apache/pulsar-test-infra/diff-only@master
+        with:
+          args: .github/workflows/ci_docker.yml
+
```

Comment:
The `apache/pulsar-test-infradiff-only` action is used here. See https://github.com/apache/pulsar-test-infra/tree/master/diff-only for more information.